src/utils/quote_report_utils.ts

```diff
@@ -101,9 +95,7 @@ export const quoteReportUtils = {
             blockNumber: logOpts.blockNumber?.toString(),
             estimatedGas: logOpts.estimatedGas.toString(),
         };
-        if (logOpts.submissionBy === 'metaTxn') {
-            logBase = { ...logBase, zeroExTransactionHash: logOpts.zeroExTransactionHash };
-        } else if (logOpts.submissionBy === 'taker') {
+        if (logOpts.submissionBy === 'taker' || logOpts.submissionBy === 'gaslessSwapAmm') {
```

Comment:
Since `logOpts` is of type `QuoteReportForTakerTxn`, then its `submissionBy` can never be `gaslessSwapAmm`. Did a quick code search and don't think we use this method in `meta_transaction`

Reply:
we're gonna kill